tools/appstreamcli.c

```diff
@@ -1340,9 +1360,14 @@ as_client_run (char **argv, int argc)
 			as_client_run_check_license);
 	ascli_add_cmd (commands,
 			2, "is-satisfied", NULL, "FILE|COMPONENT-ID",
-			/* TRANSLATORS: `appstreamcli `check-license` command description. */
+			/* TRANSLATORS: `appstreamcli `is-satisfied` command description. */
 			_("Check if requirements of a component (via its ID or MetaInfo file) are satisfied on this system."),
 			as_client_run_is_satisfied);
+	ascli_add_cmd (commands,
+		       2, "get-latest-version", NULL, "FILE",
+		       /* TRANSLATORS: `appstreamcli get-latest-version command description. */
+		       _("Get the latest version from the AppData file"),
```

Comment:
```suggestion
		       _("Get the latest version from the MetaInfo file"),
```
(the file format is called MetaInfo, AppData is an ancient name from an early draft)